flasher.cpp

```diff
@@ -723,4 +733,45 @@ void Flasher::TryToConnect()
     }
 }
 
+bool Flasher::OpenConfigFile(QJsonDocument& json_document)
+{
+    bool success = false;
+    config_file_.setFileName(kConfigFileName);
+
+    if (!config_file_.exists()) {
+        CreateDefaultConfigFile();
+    }
+
+    if (config_file_.open(QIODevice::ReadOnly)) {
+        QString json_string = config_file_.readAll();
+        config_file_.close();
+        json_document = QJsonDocument::fromJson(json_string.toUtf8());
+        if (!json_document.isEmpty()) {
+            success = true;
+        }
+    }
+
+    return success;
+}
+
+void Flasher::CreateDefaultConfigFile()
+{
+    QTextStream stream_config_file(&config_file_);
+    if (config_file_.open(QIODevice::WriteOnly)) {
+
+        QJsonDocument json_data;
+        QJsonObject json_object;
+        QJsonObject json_object_server;
+
+        json_object_server.insert("address", "141.144.224.68");
+        json_object_server.insert("port", 5322);
+        json_object_server.insert("preshared_key", "NDQ4N2Y1YjFhZTg3ZGI3MTA1MjlhYmM3");
```

Comment:
Maybe we can use constexprs for default values and place them in unnamed namespace (e.g. kDefaultPort, kDefaultAddress, kDefaultKey or something similar).